templates/dt_template/README.md

```diff
@@ -0,0 +1,58 @@
+# RHOS Deployed Topology %NEW_DT_NAME%
```

Comment:
I would remove RHOS as it's not so relevant as a DT can be technically used without.

Reply:
Done